lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -1168,6 +1262,7 @@ class ConvertTritonGPUOpToLLVMPatternBase {
   ModuleAllocation *allocation;
   IndexCacheInfo indexCacheInfo;
   mlir::triton::gpu::TMAMetadataTy *tmaMetadata;
+  std::map<std::string, int> *pStat;
```

Comment:
What does the "p" in "pStat" mean?  (This name is used in many places.)  Should this just be called "stats"?